SUI/SUI/Filters/CustomSliderView.swift

```diff
@@ -0,0 +1,97 @@
+// CustomSliderView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Кастомный слайдер
+struct CustomSliderView: View {
```

Comment:
```suggestion
struct CustomSliderView: View {
```
Custom это слово не приемлемо в названиях у Элла нет таких слов в названиях вью